CHANGELOG.md

```diff
@@ -1,5 +1,60 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.8.5
+
+<!-- generated comparing v5.8.4..master -->
+
+_Jun 20, 2022_
+
+A big thanks to the 12 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Added support for CSS variables in the `Avatar` component and the `SpeedDialAction` component respectively by @vicasas and @gin1314
+- many other 🐛 bug fixes and 📚 documentation improvements
+
+### `@mui/material@5.8.5`
+
+- &#8203;<!-- 23 -->[Avatar] Add support for CSS variables (#32499) @vicasas
+- &#8203;<!-- 15 -->[Dialog] Fix broken styles if `maxWidth` is set to `false` (#32987) @kmurgic
+- &#8203;<!-- 03 -->[SpeedDialAction] Add support for CSS variables (#32608) @gin1314
+- &#8203;<!-- 02 -->[Tabs] Increment scroll of the minimum amount possible (#33103) @oliviertassinari
+
+### `@mui/codemod@5.8.5`
+
+- &#8203;<!-- 19 -->[codemod] Preserve comments within jss-to-tss-react (#33170) @ryancogswell
+
+### `@mui/base@5.0.0-alpha.86`
+
+- &#8203;<!-- 22 -->[BadgeUnstyled] Accept callbacks in componentsProps (#33176) @michaldudak
+- &#8203;<!-- 20 -->[ButtonUnstyled] Use useSlotProps (#33096) @michaldudak
+- &#8203;<!-- 08 -->[InputUnstyled] Use useSlotProps (#33094) @michaldudak
+- &#8203;<!-- 04 -->[ModalUnstyled] Define ownerState and slot props' types (#32901) @michaldudak
+
+### `@mui/joy@5.0.0-alpha.33`
+
+- &#8203;<!-- 07 -->[Joy] Add Checkbox documentation (#33171) @siriwatknp
+- &#8203;<!-- 06 -->[Joy] Add List documentation (#33120) @siriwatknp
+- &#8203;<!-- 05 -->[Joy] Make slider displays Joy classname (#33051) @siriwatknp
+
+### Docs
+
+- &#8203;<!-- 21 -->[blog] Fix display on Safari (#33102) @oliviertassinari
+- &#8203;<!-- 14 -->[docs] Improve Joy template UX (#33159) @siriwatknp
+- &#8203;<!-- 13 -->[docs] Update Shadow DOM guide (#33160) @cherniavskii
+- &#8203;<!-- 12 -->[docs] Fix SEO regressions (#33106) @oliviertassinari
+- &#8203;<!-- 11 -->[docs] Add job ad in table of content (#33143) @mnajdova
+- &#8203;<!-- 10 -->[docs] Add customization as a value proposition (#33014) @oliviertassinari
+- &#8203;<!-- 09 -->[examples] Add example using nextjs & @mui/styles as a starter for the migration to v5 (#33005) @mnajdova
+- &#8203;<!-- 01 -->[website] Replace Airtable with Ashby links for applying to a opened position (#33193) @DanailH
+
+### Core
```

Comment:
I can't see https://github.com/mui/material-ui/pull/33226 in the changelog. Did you updated to latest master?